xdebug/xdebug

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+## Turn Xdebug on or off
+##
+## Usage: fin xdebug [on|off]
+
+set -e
+
+case $1 in
+
+    on)
+        fin config set --env=local XDEBUG_ENABLED=1
+        fin up
+        ;;
+    off)
+        fin config set --env=local XDEBUG_ENABLED=0
+        fin up
+        ;;
+    *)
+        echo "Please tell me 'on' or 'off'."
```

Comment:
Sorry, just one more nit-picky little thing. For consistency of verbiage with other commands, can you change this to:

`"Usage: fin xdebug <on|off>"`

and add `exit 1` on the next line?